File: 백준/Bronze/28701. 세제곱의 합/세제곱의 합.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long N = Long.parseLong(sc.nextLine());

        long sumN = 0;
        long sumTN = 0;
        for (int i = 1; i <= N; i++) {
            sumN += i;
            sumTN += (i * i * i);
        }

        StringBuilder sb = new StringBuilder();
        sb.append(sumN).append("\n");
        sb.append(sumN * sumN).append("\n");
        sb.append(sumTN);
        System.out.println(sb);
    }
}

File: 백준/Bronze/28701. 세제곱의 합/세제곱의 합.py
N = int(input())
sum_N = 0
sum_tN = 0
for i in range(1, N + 1):
    sum_N += i
    sum_tN += i * i * i
print(sum_N)
print(sum_N * sum_N)
print(sum_tN)